packages/probing-frontend/src/api/monitoring/monitoring.service.ts

```diff
@@ -0,0 +1,30 @@
+import axiosInstance from '@/config/axios'
+import type { EService } from '@/api/monitoring/monitoring.models'
+import type { FilterOption } from '@pagopa/interop-fe-commons'
+
+async function getList(params?: { limit: number; offset: number }): Promise<EService> {
+  const response = await axiosInstance.get<EService>(
+    `${import.meta.env.VITE_BASE_PATH}/eservices`,
```

Comment:
Could you export `import.meta.env.VITE_BASE_PATH` from the constants file as `API_BASE_PATH`?